laws/applicative.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const {identity, apply} = require('fantasy-combinators');
+const {ap} = require('../index');
+
+const id = (t) => (eq) => (x) => {
+    const a = t.of(identity).ap(t.of(x));
```

Comment:
Didn't know that! OK will change

On Wed, 27 Jan 2016, 19:08 Roman Pominov <notifications@github.com> wrote:

> In laws/applicative.js
> <https://github.com/fantasyland/fantasy-land/pull/123#discussion_r51033018>
> :
>
> > @@ -0,0 +1,24 @@
> > +'use strict';
> > +
> > +const {identity, apply} = require('fantasy-combinators')
>
> ;
> > +const {ap} = require('../index');
> > +
> > +const id = (t) => (eq) => (x) => {
> > +    const a = t.of(identity).ap(t.of(x));
>
> This should be t[of](identity)[ap](t[of](x)) I think.
>
> Btw we have a problem with of here, it's a reserved word, we can't use it
> for variable name [image: :crying_cat_face:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fantasyland/fantasy-land/pull/123/files#r51033018>.
>